qcodes/instrument_drivers/Keysight/keysight_e4980a.py

```diff
@@ -293,6 +320,59 @@ def _set_measurement(self,
         self._measurement_pair = measurement_pair
         self.write(f":FUNCtion:IMPedance {measurement_pair.name}")
 
+    def _enable_dc_bias(self, val: str) -> None:
+        """
+        Enable or disable DC bias.
+        """
+        if self.option_number() != "001":
+            raise RuntimeError("Option 001 is not installed. Hence, DC Bias "
+                               "cannot be enabled/disabled.")
+
+        self.write(f":BIAS:STATe {val}")
+
+    def _get_dc_bias_state(self) -> str:
+        """
+        Gets the DC bias state.
+        """
+        if self.option_number() != "001":
+            raise RuntimeError("Option 001 is not installed. Hence, DC Bias "
+                               "state cannot be determined.")
+
+        return self.ask(":BIAS:STATe?")
+
+    def _set_dc_bias_voltage_level(self, val: float) -> None:
+        """
+        Sets DC bias voltage level.
+        """
+        if self.option_number() != "001":
+            raise RuntimeError("Option 001 is not installed. Hence, DC Bias "
+                               "voltage level cannot be set.")
+
+        self.write(f":BIAS:VOLTage[:LEVel] {val}")
+
+    def _get_dc_bias_voltage_level(self) -> float:
+        """
+        Gets DC Bias voltage level.
+        """
+        if self.option_number() != "001":
+            raise RuntimeError("Option 001 is not installed. Hence, cannot "
+                               "get DC Bias voltage level")
+
+        return float(self.ask(":BIAS:VOLTage[:LEVel]?"))
+
+    def _set_measurement_time(self, duration: str, avg_rate: int) -> None:
+        """
+        Sets measurement time and averaging rate. Measurement time duration
+        can be short, medium or long.
+        """
+        self.write(f":APERture {duration},{avg_rate}")
```

Comment:
looks like a candidate for a group parameter since two values are set/get by single command